network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -30,6 +35,16 @@
     image_field = ImageChooserPanel('image')
     MEDIA_URL = settings.MEDIA_URL
 
+vote_throttle_class = UserVoteRateThrottle if not settings.TESTING else TestUserVoteRateThrottle
+
+
+class ProductPageVoteBin(models.Model):
```

Comment:
naming problem: this is not a vote bin, this is a collection of bins, so the class has to either be `ProductPageVoteBins` or `ProductPageVotes` (I'd go with the latter, because that allows future extensions without having a then-badly-named class).